src/ripple/app/misc/impl/ValidatorList.cpp

```diff
@@ -997,7 +1030,7 @@ ValidatorList::getNegativeUNL() const
 void
 ValidatorList::setNegativeUNL(hash_set<PublicKey> const& negUnl)
 {
-    std::lock_guard lock{mutex_};
+    std::unique_lock lock{mutex_};
```

Comment:
Super nitpicky:  This could be `unique_lock` (unqualified).  Only difference is readability.  I found myself wondering why the local typedefs weren't used.  Ditto 2 places above with `std::shared_lock`.

Reply:
My intention was to only use the typedefs where they were required, because the template params couldn't be determined by the compiler. I don't have a strong opinion against changing it, but I'm fine leaving it as-is.

Reply:
Fine with me either way.  Approved.  Nice detective work on this one!

Reply:
Thanks!